src/index.js

```diff
@@ -54,7 +54,7 @@ export default function (CustomElement, opts) {
           return;
         }
 
-        if (name.indexOf('on') === 0) {
+        if (name.indexOf('on') === 0 && name[2] === name[2].toUpperCase()) {
           syncEvent(node, name.substring(2), props[name]);
```

Comment:
In my earlier PR, https://github.com/webcomponents/react-integration/pull/34/files#diff-1fdf421c05c1140f6d71444ea2b27638L10 is converting the first letter to lower case.